homeassistant/components/cover/mqtt.py

```diff
@@ -233,7 +252,12 @@ def current_cover_tilt_position(self):
     @property
     def supported_features(self):
         """Flag supported features."""
-        supported_features = SUPPORT_OPEN | SUPPORT_CLOSE | SUPPORT_STOP
+        supported_features = 0
+        if self._command_topic is not None:
+            supported_features = OPEN_CLOSE_FEATURES
+
+        if self._position_topic is not None:
+            supported_features |= SUPPORT_SET_POSITION
 
         if self.current_cover_position is not None:
```

Comment:
I think this should now be removed since we're using `self._position_topic` to define `SUPPORT_SET_POSITION`. It probably shouldn't have been set before.